docs/UseOverviewGuide.md

```diff
@@ -80,15 +80,12 @@ We highly recommend using 2.2 first in the Project Manager GUI ([Option 3](docs/
 ##### *What scenario do you have?*
 
 - **I have single animal videos, but I want to use the advanced tracking features & updated network capabilities introduced (for multi-animal projects) in DLC2.2:**
-   - quick start: when you `create_new_project` just set the flag `multianimal=True`.
+   - quick start: when you `create_new_project` just set the flag `multianimal=False`. You still get lots of upgrades!
```

Comment:
this is not correct; it is written correctly. You can use PAFs with single animals, but MA must be set to TRUE.